test/postServiceTest.js

```diff
@@ -0,0 +1,38 @@
+/* global it, describe */
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const PostService = require('../services/PostService');
+
+describe('PostService', () => {
+  describe('getOnePost(post_id)', () => {
+    it('should return correct post', () => {
+      const stub = sinon.stub(Post, 'findById').returns(5);
+      const post = PostService.getOnePost(5);
+      assert.equal(post, 5);
+      stub.restore();
+    });
```

Comment:
The problem with stubbing and calling your method like this is that the following implementation will pass your tests:
```
getOnePost(postId) {
    Post.findById(postId);
    return postId;
}
```

Your assertion that the return value of `findById()` is the same as `getOnePost()`  is too weak

Reply:
fixed by having different values for getOnePost parameter and findById parameter